src/ovp_pipeline/commands/source_coverage.py

```
@@ -0,0 +1,308 @@
+        if handle is None or handle in known_authors:
+            continue
+        handle_counts[handle] += 1
+    return sorted(handle_counts.items(), key=lambda kv: -kv[1])
+
+
+def _load_known_authors(vault_dir: Path) -> set[str]:
+    out: set[str] = set()
+    authors_jsonl = vault_dir / "60-Logs" / "authors.jsonl"
+    if authors_jsonl.exists():
+        for line in authors_jsonl.read_text(encoding="utf-8").splitlines():
+            line = line.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)